README.md

```diff
@@ -19,7 +19,7 @@ Partitions are generated based on segment ownership and allows processing of dat
  <dependency>  
     <groupId>org.infinispan.hadoop</groupId>  
     <artifactId>infinispan-hadoop-core</artifactId>  
-    <version>0.1-SNAPSHOT</version>  
+    <version>0.1</version>  
```

Comment:
yes, but release should come less than 1h after this PR is integrated, so I anticipated the work